<launch>
  <param name="initial_package" type="string" value="nao_playing_memory" />
  <param name="initial_goal" type="string" value="owl_parse('package://nao_playing_memory/owl/Nao_computables.owl')" />
  <node name="json_prolog" pkg="json_prolog" type="json_prolog_node" cwd="node" output="screen" />
</launch>
